portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -79,4 +80,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/index.html");
   }
+
+  public String translate(String text, String code){
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+    Translation translation = 
+      translate.translate(text, Translate.TranslateOption.targetLanguage(code));
+    String translatedText = translation.getTranslatedText();
+    return translatedText;
```

Comment:
This can be condensed.
```suggestion
    return translation.getTranslatedText();
```